### PR TITLE
Feature/71/enemy random movement

### PR DESCRIPTION
**Pull request by Muhammed-als**

### CODE REVIEW

**Review by MGertz — APPROVED** [review]
LGTM

File: RandomMovement/src/main/java/module-info.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    requires Common;
+    requires java.desktop;
+
+    exports dk.sdu.sesem4.RandomMovement;
+
+    provides dk.sdu.sesem4.common.SPI.MovementControllerSPI with dk.sdu.sesem4.RandomMovement.RandomMovementController;
+    provides dk.sdu.sesem4.common.SPI.ControlSPI with dk.sdu.sesem4.RandomMovement.RandomMovement;
+
+
+
```
Comment (The0mikkel) [inline RandomMovement/src/main/java/module-info.java:11]: Seems like there is a lot of empty lines just floating here?

**Review by The0mikkel — COMMENTED** [review]
There is a few things with formatting. If everyone else is alright with that, then I am too.

File: RandomMovement/src/main/java/module-info.java (hunk 1 above)
Comment (Muhammed-als) [reply RandomMovement/src/main/java/module-info.java]: I have removed the empty lines 👍

**Review by The0mikkel — APPROVED** [review]

**Review by JfriisJ — APPROVED** [review]
looks good

File: Common/src/main/java/dk/sdu/sesem4/common/data/controllerParts/ControlType.java [hunk 2]
```diff
@@ -6,5 +6,9 @@
 public enum ControlType {
     KEYBOARD,
     GAMEPAD,
-    JOYSTICK
+    JOYSTICK,
+    DUMBAI,
```
Comment (jakob753951) [inline Common/src/main/java/dk/sdu/sesem4/common/data/controllerParts/ControlType.java:10]: underscores mellem ord?

**Review by jakob753951 — CHANGES_REQUESTED** [review]
close, but no cigar

**Review by jakob753951 — APPROVED** [review]
LGTM

**Review by MGertz — APPROVED** [review]
JUST MERGE THIS SHIT

**Review by The0mikkel — APPROVED** [review]

**Pull request merged**